tests/bitcoin_v2.rs

```diff
@@ -233,13 +262,19 @@ fn bitcoin_v2_reverse() {
     // Give a valid claim address or else funds will be lost.
     let claim_address = "tb1qq20a7gqewc0un9mxxlqyqwn7ut7zjrj9y3d0mu".to_string();
 
+    // Sign the address signature by claim priv key.
+    let address_hash = sha256::Hash::hash(&Vec::from_hex(&claim_address).unwrap());
+    let msg = Message::from_digest_slice(address_hash.as_byte_array()).unwrap();
+    let addrs_sig = Secp256k1::new().sign_schnorr(&msg, &our_keys);
+
     let create_reverse_req = CreateReverseRequest {
         invoice_amount,
         from: "BTC".to_string(),
         to: "BTC".to_string(),
         preimage_hash: preimage.sha256,
+        address_signature: addrs_sig.to_string(),
```

Comment:
You also need to pass the address itself

Reply:
Done.